src/Compilers/CSharp/Portable/CSharpExtensions.cs

```diff
@@ -6,29 +6,52 @@
 using System.Threading;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
-using Microsoft.CodeAnalysis.CSharp.Symbols;
-using Microsoft.CodeAnalysis.Syntax;
 using Microsoft.CodeAnalysis.Operations;
+using Microsoft.CodeAnalysis.Syntax;
 
 namespace Microsoft.CodeAnalysis
 {
     public static class CSharpExtensions
     {
+        /// <summary>
+        /// Determines if <see cref="SyntaxToken"/> is of a specified kind.
+        /// </summary>
+        /// <param name="token">The source token.</param>
+        /// <param name="kind">The syntax kind to test for.</param>
+        /// <returns><c>true</c> if token is of specified kind; <c>false</c> otherwise.</returns>
```

Comment:
```suggestion
        /// <returns><see langword="true"/> if the token is of the specified kind; otherwise, <see langword="false"/>.</returns>
```

Reply:
Thanks for the improvement! I wasn't sure which syntax was the correct one and it seems that SO answers are wrong xD